### PR TITLE
5.2.0

### PR DESCRIPTION
**Pull request by rekmarks**
- `5.2.0`
- Update changelog
- Fix Git URLs in `package.json`
- Delete `babel.config.json`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
+## [Unreleased]
 
+## [5.2.0] - 2020-07-24
+
+### Changed
+
+- Remove `async` and `promise-to-callback` dependencies
+  - These dependencies were used internally for asynchronous control flow.
+  They have been replaced with Promises and native `async`/`await`.
+  This has introduced subtle timing differences during middleware execution, but none that should affect consumers.
```
Comment (Gudahtt) [inline CHANGELOG.md]: Is this still true? 🤔

**Review by Gudahtt — COMMENTED** [review]
#55 seems worth mentioning in the changelog as well, as an addition.

File: CHANGELOG.md (hunk 1 above)
Comment (rekmarks) [reply CHANGELOG.md]: The timing differences? I believe so; overall, it appears to be faster in terms of event loop ticks.

Comment (rekmarks) [reply CHANGELOG.md]: That's why e.g. https://github.com/MetaMask/metamask-extension/pull/9064 had to happen.

**Comment by rekmarks** [comment]
Added #55 to changelog in eb36d55. Also updated explanation of timing differences, and deleted `babel.config.js`, which had been missed or reintroduced during merging/rebasing.

File: CHANGELOG.md (hunk 1 above)
Comment (Gudahtt) [reply CHANGELOG.md]: Ah I see, OK. I thought that was a result of some previous iteration of the async/await branch.

**Comment by Gudahtt** [comment]
It would be nice to get #60 in first, so that the lockfile for v5.20 is free of extraneous entries.

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Pull request merged**